tensorflow/stream_executor/rocm/rocm_driver.cc

```diff
@@ -524,15 +517,6 @@ GpuDriver::ContextGetSharedMemConfig(GpuContext* context) {
                                                       GpuStreamHandle stream) {
   ScopedActivateContext activation{context};
   void* pointer = absl::bit_cast<void*>(location);
-
-  // FIXME - need to set a 32-bit value here
-  unsigned char valueC = static_cast<unsigned char>(value);
-  uint32_t value32 = (valueC << 24) | (valueC << 16) | (valueC << 8) | (valueC);
```

Comment:
same comment as before...do we need to add this logic for the call to hipMemsetAsync at line 511